lib/routes/cast/index.ts

```diff
@@ -1,71 +1,77 @@
+                        title: title.text(),
+                        link: title.attr('href'),
+                        description: '',
+                    };
+                }
+                articleUrl = `${baseUrl}${title.attr('href')}`;
+
+                return cache.tryGet(articleUrl, async () => {
+                    const res = (await got.get(articleUrl!)) as ResponseData<string>;
+                    const article = load(res.data);
+                    const pubDate = timezone(parseDate(article('meta[name=PubDate]').attr('content')!, 'YYYY-MM-DD HH:mm'), +8);
+
+                    return {
+                        title: title.text(),
+                        pubDate,
+                        description: article('#zoom').html(),
+                        link: `${baseUrl}${title.attr('href')}`,
+                    };
+                });
+            })
+    );
+}
+
 export default async (ctx) => {
-    const { column = 457 } = ctx.req.param();
-    const { limit = 10 } = ctx.req.query();
-    const link = `${baseUrl}/col/col${column}/index.html`;
-    const { data: response } = await got.post(`${baseUrl}/module/web/jpage/dataproxy.jsp`, {
-        searchParams: {
-            startrecord: 1,
-            endrecord: limit,
-            perpage: limit,
-        },
-        form: {
-            col: 1,
-            appid: 1,
-            webid: 1,
-            path: '/',
-            columnid: column,
-            sourceContentType: 1,
-            unitid: 335,
-            webname: '中国科学技术协会',
-            permissiontype: 0,
-        },
-    });
+    const { column, subColumn, category } = ctx.req.param();
+    const { limit = 20 } = ctx.req.query();
+    let link = `${baseUrl}/${column}/${subColumn}`;
+    if (category) {
+        link += `/${category}/index.html`;
+    }
+    const { data: indexData } = (await got.get(link)) as ResponseData<string>;
 
-    const $ = load(response, {
-        xml: {
-            xmlMode: true,
-        },
-    });
+    const $ = load(indexData);
 
-    const pageTitle = await cache.tryGet(link, async () => {
-        const { data: response } = await got(link);
-        const $ = load(response);
-        return $('head title').text();
-    });
+    let items: any[] = [];
 
-    const list = $('record')
-        .toArray()
-        .map((item) => {
-            item = load($(item).html(), null, false);
-            const a = item('a').first();
-            return {
-                title: a.text(),
-                pubDate: parseDate(item('.list-data').text().trim(), 'DDYYYY/MM'),
-                link: `${baseUrl}${a.attr('href')}`,
-            };
-        });
+    // 新闻-视频首页特殊处理
+    if (column === 'xw' && subColumn === 'SP' && !category) {
+        items = await parsePage(indexData);
+    } else {
+        const buildUnitScript = $('script').filter((_, el) => $(el).attr('parsetype') === 'bulidstatic');
+        const queryUrl = `${baseUrl}/${buildUnitScript.attr('url')}`;
```

Comment:
Duplicated slash
https://github.com/DIYgod/RSSHub/actions/runs/8143752325/job/22256284076#step:12:27
```suggestion
        const queryUrl = `${baseUrl}${buildUnitScript.attr('url')}`;
```